contracts/fees_collector_contract.py

```diff
@@ -1,15 +1,33 @@
-from contracts.base_contracts import BaseBoostedContract
+from copy import copy, deepcopy
+from typing import Any, cast
+from multiversx_sdk import ApiNetworkProvider, CodeMetadata, ProxyNetworkProvider, AddressComputer, SmartContractQueryResponse
+from multiversx_sdk.network_providers.transaction_decoder import TransactionDecoder
+from contracts.contract_identities import DEXContractInterface
+from contracts.farm_contract import FarmContract
+from contracts.metastaking_contract import MetaStakingContract
+from contracts.staking_contract import StakingContract
+from utils import decoding_structures
+from utils.contract_data_fetchers import FeeCollectorContractDataFetcher
 from utils.logger import get_logger
-from utils.utils_tx import deploy, endpoint_call, upgrade_call
-from utils.utils_generic import log_step_pass, log_unexpected_args
-from utils.utils_chain import Account, WrapperAddress as Address
-from multiversx_sdk import CodeMetadata, ProxyNetworkProvider
-
+from utils.utils_tx import prepare_contract_call_tx, send_contract_call_tx, deploy, endpoint_call, upgrade_call
+from utils.utils_generic import log_step_fail, log_step_pass, log_warning, log_unexpected_args
+from utils.utils_chain import Account, WrapperAddress as Address, decode_merged_attributes, hex_to_string, log_explorer_transaction, dec_to_padded_hex, string_to_base64
+from multiversx_sdk.abi.serializer import Serializer
```

Comment:
We can't get these merged into main yet since the abi module isn't released yet and we don't have it into our requirements.txt.
We'll have to keep this PR open until a release.

Reply:
Thanks for the review. 
I will update it once abi module is released.